### PR TITLE
solution

### PR DESCRIPTION
**Pull request by savchuksavchuk**
[DEMO LINK](https://savchuksavchuk.github.io/js_employees_table_DOM/)

### CODE REVIEW

File: src/scripts/main.js [hunk 1]
```diff
@@ -1,3 +1,305 @@
...
+            <td>${person.office}</td>
+            <td>${person.age}</td>
+            <td>${person.salary}</td>
+        `;
+
+    tableBody.append(insertPerson);
+  }
+}
+
+// sorting implementation
```
Comment (liliia-livandovskaya) [inline src/scripts/main.js:41]: remove comments

File: src/scripts/main.js [hunk 2]
```diff
@@ -1,3 +1,305 @@
...
+form.innerHTML = `
+    <label>Name: 
+        <input 
+            name="name" 
+            type="text" 
+            data-qa="name" 
+            required
+        >
+    </label>
+    <label>Position:
```
Comment (liliia-livandovskaya) [inline src/scripts/main.js:123]:
```suggestion
    </label>
    
    <label>Position: 
```

File: src/scripts/main.js [hunk 3]
```diff
@@ -1,3 +1,305 @@
...
+    </label>
+    <label>Position: 
+        <input 
+            name="position" 
+            type="text" 
+            data-qa="position" 
+            required
+        >
+    </label>
+    <label>
```
Comment (liliia-livandovskaya) [inline src/scripts/main.js:131]:
```suggestion
    </label>
    
    <label>
```

File: src/scripts/main.js [hunk 4]
```diff
@@ -1,3 +1,305 @@
...
+        <select name="office" required>
+            <option>Tokyo</option>
+            <option>Singapore</option>
+            <option>London</option>
+            <option>New York</option>
+            <option>Edinburgh</option>
+            <option>San Francisco</option>
+        </select>
+    </label>
+    <label>Age:
```
Comment (liliia-livandovskaya) [inline src/scripts/main.js:142]:
```suggestion
    </label>
    
    <label>
      Age: 
```

File: src/scripts/main.js [hunk 5]
```diff
@@ -1,3 +1,305 @@
...
+    </label>
+    <label>Age: 
+        <input 
+            name="age" 
+            type="number" 
+            data-qa="age" 
+            required
+        >
+    </label>
+    <label>Salary:
```
Comment (liliia-livandovskaya) [inline src/scripts/main.js:150]:
```suggestion
    </label>
    
    <label>
      Salary: 
```

**Review by liliia-livandovskaya — CHANGES_REQUESTED** [review]
Add validation on edit and pat attention to the comments
Screenshot 2022-10-06 at 11 03 38

**Review by qoa11a — APPROVED** [review]

**Pull request closed without merging**